frontend/layouts/default.vue

```diff
@@ -157,6 +164,11 @@ export default Vue.extend({
             }
         },
     },
+    created() {
+        if (process.env.ENABLE_FILE_PROTOCOL) {
```

Comment:
Please do not reuse env variables (especially without updating their name or their description). Would you expect that setting this variable to `test123` will run some extra code?

Either use one of the latest answers from https://github.com/electron/electron/issues/2288 to build a `isElectronEnviourment` helper or come up with a different approach completely (eg inserting a js code to the page from the electron thread)